.github/actions/aqua/action.yaml

```diff
@@ -7,8 +7,8 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: aquaproj/aqua-installer@928a2ee4243a9ee8312d80dc8cbaca88fb602a91 # v2.2.0
+    - uses: aquaproj/aqua-installer@v3.0.0 # v3.0.0
```

Comment:
Please use full changeset hash.
ref. https://github.com/cybozu-go/pod-security-admission/pull/130#discussion_r1551140865

Reply:
I changed hash and regist to the application.